src/generic_search.cpp

```diff
@@ -34,29 +34,15 @@
 namespace miopen {
 namespace solver {
 
-std::size_t GetTuningIterationsMax()
-{
-    return Value(MIOPEN_DEBUG_TUNING_ITERATIONS_MAX{}, std::numeric_limits<std::size_t>::max());
-}
+std::size_t GetTuningIterationsMax() { return Value(MIOPEN_DEBUG_TUNING_ITERATIONS_MAX{}); }
 
 std::chrono::milliseconds GetTuningTimeMax()
 {
-    static const auto fallback =
-        std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::hours{2});
-    static const auto res =
-        std::chrono::milliseconds{Value(MIOPEN_TUNING_TIME_MS_MAX{}, fallback.count())};
+    static const auto res = std::chrono::milliseconds{Value(MIOPEN_TUNING_TIME_MS_MAX{})};
     return res;
 }
 
-std::size_t GetTuningThreadsMax()
```

Comment:
How are preesrving this behavior using the new semantics ?

Reply:
This became the default value of the environment variable.
src/include/miopen/generic_search_controls.hpp

Reply:
@cderb You may find this useful: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-a-permanent-link-to-a-code-snippet#linking-to-code

Reply:
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/688919cdacdffa1deec423a7c57d056736a45ab4/src/include/miopen/generic_search_controls.hpp#L41-L47